### DISCUSSION TITLE
Lol wth bro what is this ?? i am too curious... what is this all about

### DISCUSSION THREAD

**Discussion by chidam333** [hongbo-miao/hongbomiao.com#6697] [Q&A] [answered]
Is it all a joke 😂 came here from a leetcode ans of finding min in a sorted array

**Comment by hongbo-miao** [comment, marked as answer]
Hi @chidam333, sorry for the late reply! Since I noticed someone else asked a similar question, I thought I’d give a answer. This repository is a place for me to get hands-on experience after taking [some courses](https://www.credential.net/profile/hongbo-miao/wallet). If I find a technology to be high-quality and potentially useful in specific scenarios, I may add it here. It also serves as a platform for knowledge sharing. Thanks for your interest! 😉

**Reply by chidam333** [reply]
I see ! I found it curious because i expected something to be on the website.

**Reply by hongbo-miao** [reply]
I see! This repository actually started with a simple index.html file that just displayed ‘Hello’ with my name, as you can see in the [first commit](https://github.com/hongbo-miao/hongbomiao.com/commit/d35ed592bc408f9c1495fd79cf2bccf019800b44). Originally, it was just my personal website, but over time, I began using it as a space to get hands-on experience after completing various courses. The website is still part of the repo, and if you click ‘Making magic happen’ on the site, it redirects here. So now, it’s more of an entry point into this repository. ☺️
